mypyc/ir/rtypes.py

```diff
@@ -279,10 +284,22 @@ def is_short_int_rprimitive(rtype: RType) -> bool:
     return rtype is short_int_rprimitive
 
 
+def is_c_int32_rprimitive(rtype: RType) -> bool:
+    return rtype is c_int32_rprimitive
+
+
+def is_c_int64_rprimitive(rtype: RType) -> bool:
+    return rtype is c_int64_rprimitive
+
+
 def is_c_int_rprimitive(rtype: RType) -> bool:
     return rtype is c_int_rprimitive
 
 
+def is_c_pyssize_t_rprimitive(rtype: RType) -> bool:
```

Comment:
I wonder if we can leave out the `is_<type>` functions for the aliased types? Maybe only having them defined for `int32` and `int64` would be sufficient?

Reply:
for now at least we need `is_c_int_rprimitive` as we've used that in `emitfunc.py`.